src/app/lib/config/toggles/index.js

```diff
@@ -18,6 +18,9 @@ const toggles = {
     radioSchedule: {
       enabled: true,
     },
+    styIncludes: {
```

Comment:
Following conversation at refinement today, I think we agreed on a singular naming policy (like `socialEmbed`).

Reply:
this was done prior to the refinement. will refactor accordingly